ci_scripts/cppwin_tflite-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
This is the script for testing with the test model, it looks good, but I would have expected also a `prod` alternative?